calico/felix/endpoint.py

```diff
@@ -438,27 +438,8 @@ def _get_endpoint_rules(endpoint_id, suffix, ip_version, local_ips, mac,
     to_chain_name, from_chain_name = chain_names(suffix)
 
     # First build the chain that manages packets to the interface.
-    # Common chain prefixes.  Allow IPv6 ICMP and conntrack rules.
-    to_chain = ["--flush %s" % to_chain_name]
-    if ip_version == 6:
-        #  In ipv6 only, there are 6 rules that need to be created first.
-        #  RETURN ipv6-icmp anywhere anywhere ipv6-icmptype 130
-        #  RETURN ipv6-icmp anywhere anywhere ipv6-icmptype 131
-        #  RETURN ipv6-icmp anywhere anywhere ipv6-icmptype 132
-        #  RETURN ipv6-icmp anywhere anywhere ipv6-icmp router-advertisement
-        #  RETURN ipv6-icmp anywhere anywhere ipv6-icmp neighbour-solicitation
-        #  RETURN ipv6-icmp anywhere anywhere ipv6-icmp neighbour-advertisement
-        #
-        #  These rules are ICMP types 130, 131, 132, 134, 135 and 136.
-        for icmp_type in ["130", "131", "132", "134", "135", "136"]:
-            to_chain.append("--append %s --jump RETURN "
```

Comment:
I don't think these are needed any more.  only the router should be sending these in a way that bypasses the incoming policy of the VM, and the router is the host so its packets only hit the OUTPUT chain anyway.